### DISCUSSION TITLE
Grove - Barometer Sensor(BME280) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#619] [Q&A]
# Grove - Barometer Sensor(BME280) | Seeed Studio Wiki

Grove - Barometer Sensor(BME280)

https://wiki.seeedstudio.com/Grove-Barometer_Sensor-BME280/

**Comment by scottmclesly** [comment]
The example sketch throws same readings of:
13:14:48.029 -> Temp: 0.00C
13:14:48.029 -> Pressure: 0Pa
13:14:48.029 -> Altitude: 44330.76m
13:14:48.029 -> Humidity: 0.00% 

Any Suggestions??
